CMakeLists.txt

```diff
@@ -99,10 +100,8 @@ find_python_module(sympy 1.2 REQUIRED)
 find_python_module(textwrap 0.9 REQUIRED)
 find_python_module(yaml 3.12 REQUIRED)
 
-include_directories(${NMODL_PROJECT_SOURCE_DIR}
-                    ${NMODL_PROJECT_SOURCE_DIR}/src
-                    ${PROJECT_BINARY_DIR}/src
-                    ${NMODL_PROJECT_SOURCE_DIR}/ext)
+include_directories(${NMODL_PROJECT_SOURCE_DIR} ${NMODL_PROJECT_SOURCE_DIR}/src
+                    ${PROJECT_BINARY_DIR}/src ${NMODL_PROJECT_SOURCE_DIR}/ext)
 
```

Comment:
@tristan0x  : It's not relevant here but just to mention that the previous formatting was looking better (at least to me).